src/test/netbase_tests.cpp

```diff
@@ -427,14 +433,23 @@ BOOST_AUTO_TEST_CASE(netpermissions_test)
 
     // Happy path for whitelist parsing
     BOOST_CHECK(NetWhitelistPermissions::TryParse("noban@1.2.3.4", whitelistPermissions, error));
-    BOOST_CHECK_EQUAL(whitelistPermissions.m_flags, PF_NOBAN);
+    BOOST_CHECK_EQUAL(whitelistPermissions.m_flags, NetPermissionFlags::NoBan);
+    BOOST_CHECK(NetPermissions::HasFlag(whitelistPermissions.m_flags, NetPermissionFlags::NoBan));
+    BOOST_CHECK(NetPermissions::HasAnyFlag(whitelistPermissions.m_flags, NetPermissionFlags::NoBan));
+
+    NetPermissions::ClearFlag(whitelistPermissions.m_flags, NetPermissionFlags::Download);
+    BOOST_CHECK(!NetPermissions::HasFlag(whitelistPermissions.m_flags, NetPermissionFlags::Download));
+    BOOST_CHECK(!NetPermissions::HasFlag(whitelistPermissions.m_flags, NetPermissionFlags::NoBan));
```

Comment:
Maybe move this line one line below? So that it is clear how `HasFlag` and `HasAnyFlag` differ.

Reply:
sure, done